src/hooks/useDomainsData.ts

```diff
@@ -0,0 +1,69 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+
+import { useGetDomainsQuery } from '../generated/graphql/ens';
+
+interface DomainDataType {
+  id: string;
+  name?: string;
+  labelName?: string;
+  data?: any;
+}
+
+export function useDomainsData(address: string) {
+  const [mainLoader, setMainLoader] = useState(false);
+  const [domainArray, setDomainArray] = useState<DomainDataType[]>([]);
+  const [emptyDomainArray, setEmptyDomainArray] = useState<DomainDataType[]>([]);
+
+  const { data, loading } = useGetDomainsQuery({
+    variables: {
+      address: address.toLowerCase(),
+    },
+  });
```

Comment:
Here is piece of code where you input custom address for testing

Reply:
Use [impersonator](https://www.impersonator.xyz/) instead.